### PR TITLE
Implement status pings

### PR DESCRIPTION
**Pull request by NobodyNada**
A status ping is done by POSTing a `key` to `/status.json`.  It returns a JSON object, containing a boolean called `should_standby` which indicates whether the instance which pinged should be in standby mode.

### CODE REVIEW

File: app/models/bot.rb [hunk 1]
```diff
@@ -11,4 +11,10 @@ def eligible_collaborators
         .order(:username)
         .map{ |u| [u.username, u.id] }
   end
+
+  def active_instance
+    BotInstance.where(bot: self)
+               .where('last_ping > ?', 3.minutes.ago)
```
Comment (Undo1) [inline app/models/bot.rb]: Probably should either take the instance with the most recent seen time, or rename this to `active_instances` and return an array. With Smokey, for example, we sometimes run two active instances for testing or whatnot.

Comment (NobodyNada) [inline app/models/bot.rb]: This is supposed to be the instance that *should* be active, not the ones that are currently active.  Maybe I should rename it to `preferred_instance`?

**Comment by ArtOfCode-** [comment]
The preferred instance functionality should be manual, ideally.

**Comment by NobodyNada** [comment]
@ArtOfCode- Yeah, I agree.  I'll implement this when I have time (probably this afternoon or evening, PDT); see my messages staring [here](http://chat.stackoverflow.com/transcript/message/36073808#36073808)

**Pull request merged**